main.py

```diff
@@ -52,54 +51,73 @@ def equals(self, args):
                 return False
         return cur != ' '
 
-    def checkWinner(self):
```

Comment:
Too complex function - splitting into separate functions to increase readability

Reply:
This function **returns** the winner. Should be named correspondingly